lib/typescript/render/providers/facebook/facebookModel.ts

```diff
@@ -1,12 +1,42 @@
+export enum AttachmentType {
+  image = 'image',
+  video = 'video',
+  audio = 'audio',
+  file = 'file',
+  fallback = 'fallback',
+}
+
+export interface Attachment {
+  type:
+    | AttachmentType.image
+    | AttachmentType.video
+    | AttachmentType.audio
+    | AttachmentType.file
+    | AttachmentType.fallback;
+  payload: {
+    title?: string;
+    url?: string;
+  };
+}
+
+export enum ContentType {
+  text = 'text',
+  image = 'image',
+}
+
 export interface Content {
-  type: 'text';
+  type: ContentType.image | ContentType.text;
```

Comment:
So this is a very subtle difference, but I think I would prefer string unions here for the reason that Typescript enums actually get compiled to a Javascript snippet. This snippet we don't really need since we control the intermediary content payloads. For string unions, it'd be statically checked at compile time only.

Also this may be a question of taste, but I feel like it's easier to grasp the available file types [this way](https://github.com/airyhq/airy/blob/develop/lib/typescript/types/content.ts#L11)